packages/build/bin/build-css.js

```diff
@@ -1,18 +1,17 @@
 #!/usr/bin/env node
-
 const chalk = require('chalk')
 const program = require('commander')
 const path = require('path')
 
 const { css } = require('../index')
-const defaultInputFilePath = path.join(process.cwd(), 'dist', 'css', 'index.js')
-const defaultOutputDirectoryPath = path.join(process.cwd(), 'dist', 'css')
+const defaultInputFilePath = 'dist/css/index.js'
+const defaultOutputDirectoryPath = 'dist/'
```

Comment:
Why rm the smarter path.join?  It worked, and it was guaranteed to be cross-platform happy.

Reply:
turns out the --input flag did not work unless I did this